### PR TITLE
Math magicians: Components

### PR DESCRIPTION
**Pull request by ridaarif98**
- Delete all the boilerplate from CRA (text, images, styles)
- Create a directory called `components`.
- Inside `components`, create a new `Calculator.js` file.
- In `Calculator.js`, create a React component that matches the design
- Used class-based components.
- Import class and use it in the main component (App.js).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,9 @@
...
-import './App.css';
+import './styles/style.css';
+import CalculatorDesign from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
```
Comment (henatan99) [inline src/App.js]: - [ ] The App.js component should be class-based as per the project's requirement. In this milestone, the objective is to practice class-based React components. Kindly make changes here. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @ridaarif98 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (ridaarif98) [reply src/App.js]: Thanks for the review. But in this project requirements in Calculator.js, we need to use class.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @ridaarif98 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**